bastionzero/service/agents/agents.go

```diff
@@ -57,6 +61,8 @@ type ListAgentsOptions struct {
 	EnvironmentName string `url:"environmentName,omitempty"`
 	// Only return agents whose name contains this substring. Note that terminated agents have no name.
 	Name string `url:"name,omitempty"`
+	// Only return agents with these UUIDs. Note that terminated agents have no ID.
+	AgentIds []string `url:"agentIds,omitempty"`
```

Comment:
Create another `changie` entry of `kind: Enhancements`:

`agents: Add ability to filter by agent ID when listing agents`

or something like that ^^ for entry